book_inventory.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+
+#define LENGTH_TITLE (41)
+#define LENGTH_AUTHOR (31)
+
+char * s_gets(char *state_book, int number_of_book);
+
+struct book_price{
+    char tittle [LENGTH_TITLE];
```

Comment:
```suggestion
    char title [LENGTH_TITLE];
```

There's a spell checker extension for VS code that you can use to avoid silly mistakes like this